packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -359,8 +359,13 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 
 				await initRunners(resolvedPluginConfig, viteDevServer, miniflare);
 
+				const viteHandle =
+					/^https?:\/\/[^/]+\/(?:(?:src|node_modules[^\/]*?\/)|@\w+)|\?import$/i;
```

Comment:
I don't believe this is sufficient to handle all the scenarios where the Worker should not intercept the request, and even if it worked now (which I don't think it would) it would be brittle to future changes to Vite and also to other plugins that are used alongside this one.